lib/susedistribution.pm

```diff
@@ -277,21 +277,20 @@ sub activate_console {
             $nr = 2 if ($user eq 'root');
             # we need to wait more than five seconds here to pass the idle timeout in
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
-            assert_screen "tty$nr-selected";
+            assert_screen "tty$nr-selected", 60;
 
             assert_screen "text-login";
             type_string "$user\n";
             if (!get_var("LIVETEST")) {
                 assert_screen "password-prompt";
                 type_password;
-                type_string "\n";
             }
         }
         else {
             # different console-behaviour for s390x
             $self->script_run("su - $user") unless ($user eq 'root');
         }
-        assert_screen "text-logged-in-$user", 10;
+        send_key_until_needlematch "text-logged-in-$user", "ret", 6, 5;
```

Comment:
the alt-f1 case is something I can see human testers doing too, but here I wonder what went wrong - and if there aren't smarter ways to handle it than brute force.